### PR TITLE
✏️(project) fix non-critical typos

### PR DESCRIPTION
**Pull request by lebaudantoine**
Found typos, and fixed them.

### CODE REVIEW

File: src/ralph/api/auth/oidc.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def get_authenticated_user(
     """Decode and validate OpenId Connect ID token against issuer in config.
 
     Args:
-        token (str): Base64 encoded OIDC Token. This is invoked behind the
+        auth_header (str): Base64 encoded OIDC Token. This is invoked behind the
```
Comment (lebaudantoine) [inline src/ralph/api/auth/oidc.py:101]: I am not sure the description is up-to-date.

File: tests/fixtures/auth.py [hunk 2]
```diff
@@ -77,12 +77,11 @@ def auth_credentials(fs, user_scopes=None, agent=None):
     Args:
         fs: fixture provided by pyfakefs (not called in the code)
         user_scopes (List[str]): list of scopes to associate to the user
+        agent (dict): valid Agent (per xAPI specification) representing the user
 
     Returns:
         credentials (str): auth parameters that need to be passed
             through headers to authenticate the request.
-        user_scopes (List[str]): list of scopes for the created user
-        agent (dict): valid Agent (per xAPI specification) representing the user
```
Comment (lebaudantoine) [inline tests/fixtures/auth.py:85]: what is it the right update ?

File: docs/backends.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ in most of its commands.
 Each backend has its own parameters that are required to use it. Those
 parameters can be set as command line options or environment variables; the
 later is the **recommended solution** for sensible data such as service
-credentials. If we consider the `os_username` (OpenStack user name) parameter
+credentials. If we consider the `os_username` (OpenStack username) parameter
```
Comment (Leobouloc) [inline docs/backends.md:18]:
```suggestion
credentials. If we consider the `os_username` (OpenStack user name) parameter
```

"user name" seems to be the way they present it here:

https://docs.openstack.org/python-openstackclient/pike/cli/command-objects/user.html

File: tests/api/test_statements_get.py [hunk 4]
```diff
@@ -144,7 +144,7 @@ def create_mock_agent(
 def insert_statements_and_monkeypatch_backend(
     request, es, mongo, clickhouse, monkeypatch
 ):
-    """Retuns a function that inserts statements into each backend."""
+    """Returns a function that inserts statements into each backend."""
```
Comment (Leobouloc) [inline tests/api/test_statements_get.py:147]:
```suggestion
    """Return a function that inserts statements into each backend."""
```

We use imperative in Ralph

**Review by wilbrdt — APPROVED** [review]
Alt text
LGTM :tada:

File: tests/backends/storage/test_ldp.py [hunk 5]
```diff
@@ -203,7 +203,7 @@ def mock_list(url):
 def test_backends_storage_ldp_list_method_history_management(
     monkeypatch, fs, settings_fs
 ):
-    """Tests the LDPStorage list method with an history."""
+    """Tests the LDPStorage list method with a history."""
```
Comment (Leobouloc) [inline tests/backends/storage/test_ldp.py:206]:
```suggestion
    """Test the LDPStorage list method with a history."""
```

File: tests/models/edx/test_base.py [hunk 6]
```diff
@@ -44,7 +44,7 @@ def test_models_edx_base_edx_model_with_valid_statement(statement):
 )
 @custom_given(BaseEdxModel)
 def test_models_edx_base_edx_model_with_invalid_statement(course_id, error, statement):
-    """Tests that a invalid base `Edx` statement raises a `ValidationError`."""
+    """Tests that an invalid base `Edx` statement raises a `ValidationError`."""
```
Comment (Leobouloc) [inline tests/models/edx/test_base.py:47]:
```suggestion
    """Test that an invalid base `Edx` statement raises a `ValidationError`."""
```

File: tests/models/edx/test_browser.py [hunk 7]
```diff
@@ -32,7 +32,7 @@ def test_models_edx_base_browser_model_with_valid_statement(statement):
 def test_models_edx_base_browser_model_with_invalid_statement(
     session, error, statement
 ):
-    """Tests that a invalid base browser statement raises a `ValidationError`."""
+    """Tests that an invalid base browser statement raises a `ValidationError`."""
```
Comment (Leobouloc) [inline tests/models/edx/test_browser.py:35]:
```suggestion
    """Test that an invalid base browser statement raises a `ValidationError`."""
```

File: tests/models/xapi/base/test_unnested_objects.py [hunk 8]
```diff
@@ -39,7 +39,7 @@ def test_models_xapi_base_object_interaction_component_with_valid_field(
 def test_models_xapi_base_object_interaction_component_with_invalid_field(
     id_value, field
 ):
-    """Tests a invalid `id` property in
+    """Tests an invalid `id` property in
```
Comment (Leobouloc) [inline tests/models/xapi/base/test_unnested_objects.py:42]:
```suggestion
    """Test an invalid `id` property in
```

File: tests/models/xapi/test_navigation.py [hunk 9]
```diff
@@ -26,7 +26,7 @@ def test_models_xapi_navigation_selectors_with_valid_statements(class_, data):
 
 @custom_given(PageTerminated)
 def test_models_xapi_navigation_page_terminated_with_valid_statement(statement):
-    """Tests that a valide page_terminated statement has the expected `verb`.`id` and
+    """Tests that a valid page_terminated statement has the expected `verb`.`id` and
```
Comment (Leobouloc) [inline tests/models/xapi/test_navigation.py:29]:
```suggestion
    """Test that a valid page_terminated statement has the expected `verb`.`id` and
```

**Review by Leobouloc — APPROVED** [review]
Very satisfying PR !!

File: src/ralph/api/auth/oidc.py (hunk 1 above)
Comment (wilbrdt) [reply src/ralph/api/auth/oidc.py:101]: Oh indeed, missed it in my review!
```suggestion
        auth_header (str): Authentication header containing the Base64 encoded OIDC Token. This is invoked behind the
```

File: src/ralph/models/selector.py [hunk 10]
```diff
@@ -22,7 +22,7 @@ class LazyModelField:
     path: Tuple[str]
 
     def __init__(self, path: str):
-        """Initalizes Lazy Model Field."""
+        """Initializes Lazy Model Field."""
```
Comment (quitterie-lcs) [inline src/ralph/models/selector.py:25]:
```suggestion
        """Initialize Lazy Model Field."""
```

File: tests/fixtures/auth.py (hunk 2 above)
Comment (lebaudantoine) [reply tests/fixtures/auth.py:85]: @wilbrdt WDYT ?

File: docs/backends.md (hunk 3 above)
Comment (lebaudantoine) [reply docs/backends.md:18]: The variable is, in fact, named `os_username`, not `os_user-name`.
I fixed up your comment.

**Review by Leobouloc — APPROVED** [review]
Perfect.

**Pull request merged**